tracr/io/read.py

```diff
@@ -1,11 +1,41 @@
 # tries to guess the format of the file
+"""
+Read in TIF or DCM intensity data and send to appropriate reader depending on
+format. Assume that a folder input contains data frames directly below
+(i.e. there are no nested folders.)
+
+INPUT:
+    - TIF: file (single or multilayer), or folder of TIF frames
+    - DICOM: file (single), or folder of DCM frames
```

Comment:
Same comment as above. We just want to make sure this is clearly marked. Should we include an `order` keyword and perhaps an enumeration, e.g. `order=tracr.io.BOTTOM_TO_TOP` for sequential files collected from bottom to top?